tutorial/train_horizon_net/README.md

```diff
@@ -4,18 +4,26 @@ In this tutorial, we will walk you through how to train your own model using pil
 
 ### Step 1: download pre-trained models
 ```bash
+# use -h for more details
 python download_pretrained_ckpt.py
 # or specify download info and output directory
-python download_pretrained_ckpt.py -f {DOWNLOAD_INFO} -o {OUTPUT_DIR}
+python download_pretrained_ckpt.py -f $DOWNLOAD_INFO -o $OUTPUT_DIR
```

Comment:
We are using this format ${ENV_VAR}

Reply:
Usually, "{}" is only used when trying to concatenate variables.